<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGest</class>
 <widget class="QMainWindow" name="QtGest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Logout"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Tile_Windows"/>
    <addaction name="actionC_ascade_Window"/>
   </widget>
   <widget class="QMenu" name="menu_Contracte">
    <property name="title">
     <string>&amp;Contracte</string>
    </property>
    <addaction name="action_Adaugare"/>
    <addaction name="actionEditare"/>
    <addaction name="action_Finalizare"/>
    <addaction name="action_Find"/>
    <addaction name="action_Refresh"/>
    <addaction name="separator"/>
    <addaction name="actionF_inalizate"/>
   </widget>
   <widget class="QMenu" name="menu_Admin">
    <property name="title">
     <string>&amp;Admin</string>
    </property>
    <addaction name="action_Furnizori"/>
    <addaction name="action_Users"/>
    <addaction name="action_Asiguratori"/>
   </widget>
   <widget class="QMenu" name="menu_Rapoarte">
    <property name="title">
     <string>&amp;Rapoarte</string>
    </property>
    <addaction name="actionR_apoarte"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Contracte"/>
   <addaction name="menu_Rapoarte"/>
   <addaction name="menu_Admin"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Adaugare"/>
   <addaction name="actionEditare"/>
   <addaction name="action_Finalizare"/>
   <addaction name="action_Find"/>
   <addaction name="action_Refresh"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Adaugare">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/add</normaloff>:/add</iconset>
   </property>
   <property name="text">
    <string>&amp;Adaugare</string>
   </property>
   <property name="statusTip">
    <string>Adaugare Contract Nou</string>
   </property>
  </action>
  <action name="actionEditare">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/edit</normaloff>:/edit</iconset>
   </property>
   <property name="text">
    <string>&amp;Editare</string>
   </property>
   <property name="statusTip">
    <string>Editare Contract</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Finalizare">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/finalizare</normaloff>:/finalizare</iconset>
   </property>
   <property name="text">
    <string>&amp;Finalizare</string>
   </property>
  </action>
  <action name="action_Logout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/quit</normaloff>:/quit</iconset>
   </property>
   <property name="text">
    <string>&amp;Logout</string>
   </property>
  </action>
  <action name="action_Collapse_All">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/collapse</normaloff>:/collapse</iconset>
   </property>
   <property name="text">
    <string>&amp;Collapse All</string>
   </property>
  </action>
  <action name="action_Expand_All">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/expand</normaloff>:/expand</iconset>
   </property>
   <property name="text">
    <string>&amp;Expand All</string>
   </property>
  </action>
  <action name="action_Tile_Windows">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tile</normaloff>:/tile</iconset>
   </property>
   <property name="text">
    <string>&amp;Tile Windows</string>
   </property>
  </action>
  <action name="actionC_ascade_Window">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/cascade</normaloff>:/cascade</iconset>
   </property>
   <property name="text">
    <string>C&amp;ascade Window</string>
   </property>
  </action>
  <action name="action_Furnizori">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/copy</normaloff>:/copy</iconset>
   </property>
   <property name="text">
    <string>&amp;Furnizori</string>
   </property>
  </action>
  <action name="action_Users">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/user</normaloff>:/user</iconset>
   </property>
   <property name="text">
    <string>&amp;Users</string>
   </property>
  </action>
  <action name="actionR_apoarte">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rapoarte</normaloff>:/rapoarte</iconset>
   </property>
   <property name="text">
    <string>R&amp;apoarte</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/search</normaloff>:/search</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
  </action>
  <action name="action_Refresh">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/refresh</normaloff>:/refresh</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
  </action>
  <action name="actionF_inalizate">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/remove</normaloff>:/remove</iconset>
   </property>
   <property name="text">
    <string>F&amp;inalizate</string>
   </property>
  </action>
  <action name="action_Asiguratori">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rapoarte</normaloff>:/rapoarte</iconset>
   </property>
   <property name="text">
    <string>&amp;Asiguratori</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
